website/docs/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx

```diff
@@ -79,15 +79,22 @@ To edit your Azure SSO configuration settings:
 To configure team management in your Microsoft Azure AD application:
 
 1. Navigate to the single sign-on page.
-1. Edit step 2, "User Attributes & Claims."
-   We recommend naming it "MemberOf", leaving the namespace blank, and potentially sourcing `user.assignedroles` as an easy starting point.
+1. Edit step 2, "User Attributes & Claims."  
+2. Add a new group claim.
+3. In the "Group Claims" side window, select the "Security Groups" radio button.  
+   In the "Source Attribute" field, select either sAMAccountName to pass group names in the SAML or "Group ID" to pass thru the UUIDs for the group. See the related Note for more information on UUIDs.
+   Check the "Customize the name of the group claim" box. Use "MemberOf" in the "Name (required)" field. Leave the namespace field blank.
```

Comment:
A couple of questions for you:
- Is this the right name for this field? sAMAccountName. (Part of me felt like it could have been SAML, so I just wanted to make sure).
- I felt a bit confused by the phrase "pass through the UUIDs". I changed it to "use" because I think we're specifying what attribute to use, but please let me know if I misinterpreted that!
- Does the user need to click Save or anything after all of this is done? I just want to make sure this is the final step of the process.

Finally, just a couple of style nits that I fixed in my suggestion:
- We always want to use "Refer to" instead of "See", per our style guide.
- I added a few more numbered steps to break up the instructions a bit. We want each new action in the process to be a numbered step.
- We always want to use 1 for numbered steps (the website will automatically know which number to use next, and this helps make sure we don't end up going from like 1 to 3 by accident when we make edits in the future!).
- All UI elements (button names, etc.) should be in bold.
- We can just say the name of the button, so "Click **Enter**" rather than "Click the **Enter** button.

Let me know what you think of these edits/my questions. Thank you so much for fixing this for us!

```suggestion
1. Edit step 2, **User Attributes & Claims.** 
1. Add a new group claim.
1. In **Group Claims**, select **Security Groups**.  
1. In the **Source Attribute** field, select either **sAMAccountName** to use account names or **Group ID** to use group UUIDs. 
1. Check **Customize the name of the group claim**. 
1. Set **Name (required)** to "MemberOf" and leave the namespace field blank.
```